plugins/inputs/nfs/nfs.go

```diff
@@ -0,0 +1,344 @@
+package nfs
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NFS struct {
+	Iostat   bool
+	Fullstat bool
+}
+
+var sampleConfig = `
```

Comment:
also clarify what "fullstat" means exactly. Which stats does setting fullstat=true add? why might we want to set fullstat=false?